### ISSUE TITLE
Add workflow for tagging and release in GitHub

### ISSUE THREAD

**Issue by 403-html** [403-html/mochawesome-json-to-md#17]
```md
## What's Changed

[list of the PRs]

**Full Changelog**: https://github.com/403-html/mochawesome-json-to-md/compare/prev...next

```

**Comment by 403-html** [comment]
idea:

```yml
on:
  push:
    branches:
    - main

name: Bump Version and Create Release

jobs:
  bump_version:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Bump Version
      run: |
        if [ -n "${{ github.event.label }}" ]; then
          if [ "${{ github.event.label }}" == "major" ]; then
            npx semantic-release major
          elif [ "${{ github.event.label }}" == "minor" ]; then
            npx semantic-release minor
          elif [ "${{ github.event.label }}" == "patch" ]; then
            npx semantic-release patch
          fi
        else
          npx semantic-release patch
        fi
    - name: Create Release
      uses: actions/create-release@v1
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        tag_name: ${{ github.ref }}
        release_name: Release ${{ github.ref }}
        body: |
          New ${{ github.event.label }} release! Here's what changed:
          -
          -
          -
```

**Comment by 403-html** [comment]
closed with 1.3.0-1.3.5 release
